importDBdmpFile/check_db_dump_recent.py

```diff
@@ -47,26 +51,29 @@
     # "Couldn't open/read the provided file" differently.
     try:
         with open(log_file, 'r') as log_file:
-            lines = log_file.read().splitlines()
-            last_line = lines[-1]
+            line_list = log_file.read().splitlines()
     except IOError:
         print("An error occured trying to open/read {0}".format(log_file))
         sys.exit(RETURN_CODE_CRITICAL)
 
-    # Convert the last line of the log to a timestamp.
-    # Use a inner try...except block to handle the somewhat more expected error
-    # of "the failover process ran, but something went wrong" differently.
-    try:
-        last_success = datetime.strptime(last_line, "%Y-%m-%dT%H:%M:%S%z")
-    except ValueError:
-        print("An error occured: {0}".format(last_line))
-        sys.exit(RETURN_CODE_CRITICAL)
+    for line in reversed(line_list):
+        # If OK_STRING is in the line we are looking at, we need to extract
+        # the timestamp from that line to determine when the failover
+        # process last succeeded.
+        if OK_STRING in line:
+            last_success_string = line.split(" ")[0]
```

Comment:
I'd prefer `last_success_timestamp` as it's a bit more specific and it makes it clear which part of the string you're extracting. You use `..._datetime` once you've converted it, so it's clear which is which.

Reply:
I went with `last_success_timestamp` in #21